functions/callNodeFunction/emailTemplateVariables.js

```
@@ -212,6 +232,29 @@ export function createVariableReplacer(data, formatDate) {
+        const description = addon.CourseAddonMapping?.description || '';
+        const price = addon.priceAtPurchase || 0;
+        const currency = addon.currency || 'EUR';
+        const formattedPrice = (price / 100).toFixed(2).replace('.', ',');
+        return `– ${description}: ${formattedPrice} ${currency} inkl. MwSt.`;
+      });
+      addonsHtml = '<strong>Add-ons:</strong>\n' + addonLines.join('\n');
+    }
+    result = result.replaceAll('[Enrollment:Addons]', addonsHtml);
+    
+    // Calculate and format total cost (base price + all addon prices)
+    const basePrice = data.course?.basePrice || 0;
```

Biome (2.3.13)

[error] Shouldn't redeclare 'basePrice'. Consider to delete it or rename it.

'basePrice' is defined here:


(lint/suspicious/noRedeclare)